secretmanager/api/QuickStart/README.md

```diff
@@ -0,0 +1,41 @@
+# .NET Secret Manager Sample
+
+A sample that demonstrates how to call the
+[Secret Manager API](https://cloud.google.com/secret-manager/docs) from C#.
+
+This sample requires [.NET Core 2.1](
+    https://www.microsoft.com/net/core) or later.
+
+## Build and Run
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/dotnet/docs/setup).**
```

Comment:
Yup - that's totally valid markdown (and markdown will do the right numbering when rendered 😄). It's a huge timesaver instead of trying to rename and shift numbers.

In fact, you can also do something like this:

```
1.  a
7.  b
2.  c
1005.  d
```

and it will render:

1.  a
7.  b
2.  c
1005.  d